### PR TITLE
feat: add entry about IT

### LINKED ISSUE #3029: [Entry] Subject: Information Technology
### Reason/inspiration (optional)

I've been studying the fundamentals of tech support and noticed that the Docs project does not have an entry on Information Technology, so I would like to add an entry on this topic.  

### Entry Type (select all that apply)

New Topic Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by CBID2**
### Description
This PR adds a new file, "information-technology.md", which provides an overview of Information Technology (IT) and gives people an idea of other sectors in the tech industry that they can pursue. 

Closes #3029

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by CBID2** [comment]
Hey @SSwiniarski! :) I'm ready for a review

File: content/information-technology/information-technology.md
Comment (SSwiniarski) [inline content/information-technology/information-technology.md:1]: I have two main thoughts looking at this entry:

1. This should be a concept under the General topic, not a topic in its own right.
2. It should be longer and go into more depth. At least have a full paragraph or two.

**Comment by CBID2** [comment]
> @CBID2 I have a couple of overall comments I'd like to see addressed, then I may have other comments later.

Hi @SSwiniarski. I used this approach after being inspired by the Cybersecurity entry. I plan on adding a section of terms at the end.

**Comment by CBID2** [comment]
> > @CBID2 I have a couple of overall comments I'd like to see addressed, then I may have other comments later.
> 
> Hi @SSwiniarski. I used this approach after being inspired by the Cybersecurity entry. I plan on adding a section of terms at the end.

@yangc95, what do you think?

File: content/information-technology/information-technology.md
Comment (CBID2) [reply content/information-technology/information-technology.md:1]: Hi @SSwiniarski! :) I thought about what you said and decided to put it as a General topic. I also made some additions to the entry. Check them out and let me know what you think.

**Comment by CBID2** [comment]
@SSwiniarski, I'm ready for a review! :)

File: content/general/information-technology/information-technology.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+Title: 'Information Technology'
+Description: 'Information Technology is the use of digital technological tools to store and turn data into useful information'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/cybersecurity'
+CatalogContent:
+  - 'introduction-to-information-technology'
+---
```
Comment (SSwiniarski) [inline content/general/information-technology/information-technology.md:7]: You still need to format the Metadata as a term entry, it's still laid out as a topic.

File: content/general/information-technology/information-technology.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+<link rel="cannonical" href="https://www.codecademy.com/courses/introduction-to-it/articles/it-overview"/>
```
Comment (SSwiniarski) [inline content/general/information-technology/information-technology.md:9]: Don't use HTML in the Markdown.

File: content/general/information-technology/information-technology.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+**Information Technology**, also known as IT, is the use of digital technological tools to store and turn data into useful information. Since we live in a technology-driven world, IT plays a major role in our lives whether its fixing a cracked screen on a phone, doing a data transfer from an old computer to another one, or repairing a PC's hardware.
```
Comment (SSwiniarski) [inline content/general/information-technology/information-technology.md:11]: Can we rewrite this without using "we" or "our"?

File: content/general/information-technology/information-technology.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Roles
+
+These are the common roles in the IT industry:
```
Comment (SSwiniarski) [inline content/general/information-technology/information-technology.md:15]:
```suggestion
These are some common roles in the IT industry:
```

File: content/general/information-technology/information-technology.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Hardware Technicians: Installs, repairs, and maintains [computer hardware](https://www.codecademy.com/resources/docs/general/computer-hardware). May also be required to troubleshoot hardware and software issues, as well as provide training and support to users on how to use the devices effectively.
+- Network Engineers: Designs,plans, and implements network infrastructures ranging from local area networks(LANs), wide area networks(WANs), and intranets for businesses and organizations. Also analyzes network traffic and discuss suggestions how to increase it to fulfill the organization/business' needs.
```
Comment (SSwiniarski) [inline content/general/information-technology/information-technology.md:18]:
```suggestion
- Network Engineers: Designs, plans, and implements network infrastructures ranging from local area networks(LANs), wide area networks(WANs), and intranets for businesses and organizations. Also analyzes network traffic and discusses suggestions how to optimize it to fulfill the organization/business' needs.
```

File: content/general/information-technology/information-technology.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+- Desktop Support Personnel: Typically provides tech support(e.g.,restoring and backing up data to mobile devices) to customers via computer or phone.
```
Comment (SSwiniarski) [inline content/general/information-technology/information-technology.md:19]:
```suggestion
- Desktop Support Personnel: Typically provides tech support (e.g. restoring and backing up data to mobile devices) to customers via computer or phone.
```
Can we include more than one example, they do more than restoring/backing up data.

File: content/general/information-technology/information-technology.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+## Certifications
+
+Some IT roles require applicants to become certified. Common types of certifications include:
```
Comment (SSwiniarski) [inline content/general/information-technology/information-technology.md:23]: No Microsoft certifications? Should include at lest the most popular one or two.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@CBID2 I did a review and left a few comments. One thing that's important, you need to update the metadata now that it's no longer a topic.

**Comment by CBID2** [comment]
I made the necessary changes @SSwiniarski

File: content/general/information-technology/information-technology.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Information Technology'
+Description: 'Information Technology is the use of digital technological tools to store and turn data into useful information'
+Subjects:
+  - 'Information Technology'
```
[the remark this answers is not in the document]
Comment (CBID2) [reply content/general/information-technology/information-technology.md:5]: The documentation mentioned that I can add a new one @SSwiniarski.

File: content/general/information-technology/information-technology.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+  - 'Networking'
+Tags:
+  - 'information technology'
```
[the remark this answers is not in the document]
Comment (CBID2) [reply content/general/information-technology/information-technology.md:8]: The documentation mentions that I can add a new one @SSwiniarski.

File: content/general/information-technology/information-technology.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (SSwiniarski) [reply content/general/information-technology/information-technology.md:5]: > The documentation mentioned that I can add a new one @SSwiniarski.

@CBID2 Then you need to add the term to the subjects file in this PR.

File: content/general/information-technology/information-technology.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (SSwiniarski) [reply content/general/information-technology/information-technology.md:8]: > The [documentation](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) mentions that I can add a new one @SSwiniarski

Then you need to add the term to the Tags file in this PR. (And capitalize it.)

**Comment by CBID2** [comment]
Made the changes @SSwiniarski

**Review by SSwiniarski — APPROVED** [review]
@CBID2 Looks good for a second review.

File: content/general/information-technology/information-technology.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+**Information Technology**, also known as IT, is the use of digital technological tools to store and turn data into useful information. Due to living in a technology-driven world, IT plays a major role in people's lives whether it's fixing a cracked screen on a phone, doing a data transfer from an old computer to another one, or repairing a PC's hardware.
+
+## Roles
+
+These are some common roles in the IT industry:
+
+- Hardware Technicians: Installs, repairs, and maintains [computer hardware](https://www.codecademy.com/resources/docs/general/computer-hardware). May also be required to troubleshoot hardware and software issues, as well as provide training and support to users on how to use the devices effectively.
+- Network Engineers: Designs, plans, and implements network infrastructures ranging from local area networks(LANs), wide area networks(WANs), and intranets for businesses and organizations. Also analyzes network traffic and discusses suggestions how to optimize it to fulfill the organization/business' needs.
+- Desktop Support Personnel: Typically provides tech support (e.g. restoring and backing up data to mobile devices) to customers via computer or phone.
```
Comment (KTom101) [inline content/general/information-technology/information-technology.md:22]:
```suggestion
- Desktop Support Personnel: Typically provides tech support (e.g. restoring and backing up data to mobile devices) to customers via computer or phone. Handles a range of technical issues, including hardware and software problems.
```

File: content/general/information-technology/information-technology.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+  - 'Information Technology'
+  - 'Networking'
+Tags:
+  - 'Information Technology'
+CatalogContent:
+  - 'introduction-to-information-technology'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Information Technology**, also known as IT, is the use of digital technological tools to store and turn data into useful information. Due to living in a technology-driven world, IT plays a major role in people's lives whether it's fixing a cracked screen on a phone, doing a data transfer from an old computer to another one, or repairing a PC's hardware.
```
Comment (KTom101) [inline content/general/information-technology/information-technology.md:14]: I think we should broaden the definition a little bit more. Feel free to change the wording. 
```suggestion
**Information Technology**, also known as IT, is the use of digital tools and technologies to store, process, obtain, transmit, and manage useful information. IT encompasses a spectrum of tasks, including designing, implementing, and maintaining systems, networks, databases, and applications. Due to living in a technology-driven world, IT plays a major role in people's lives whether it's fixing a cracked screen on a phone, doing a data transfer from an old computer to another one, or repairing a PC's hardware.
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@CBID2 I've added some changes!

**Comment by CBID2** [comment]
> @CBID2 I've added some changes!

@KTom101, I made the changes! :)

**Review by KTom101 — APPROVED** [review]
@CBID2 Looks Good

**Comment by KTom101** [comment]
@CBID2 There is one additional change needed so the check passes. The file path needs to be changed so it's located in the general concepts folder.

**Comment by CBID2** [comment]
> @CBID2 There is one additional change needed so the check passes. The file path needs to be changed so it's located in the general concepts folder.

Hi @KTom101! :) I'm confused on what to change. The IT folder is in the General concepts one.

**Comment by KTom101** [comment]
@CBID2 The current path is `content/general/information-technology/information-technology.md` which is causing one of the checks to fail. 

It needs to be `content/general/concepts/information-technology/information-technology.md` adding in the "concepts" folder which is where all the other entries are located.

**Comment by CBID2** [comment]
> @CBID2 The current path is `content/general/information-technology/information-technology.md` which is causing one of the checks to fail. 
> 
> It needs to be `content/general/concepts/information-technology/information-technology.md` adding in the "concepts" folder which is where all the other entries are located. 

Ok, I think I got it @KTom101. :)

**Comment by KTom101** [comment]
> > @CBID2 The current path is `content/general/information-technology/information-technology.md` which is causing one of the checks to fail. 
> > 
> > It needs to be `content/general/concepts/information-technology/information-technology.md` adding in the "concepts" folder which is where all the other entries are located. 
> 
> Ok, I think I got it @KTom101. :)

Let me know if you have any trouble! I could always make the change on my end.

**Comment by CBID2** [comment]
> > > @CBID2 The current path is `content/general/information-technology/information-technology.md` which is causing one of the checks to fail. 
> > > 
> > > It needs to be `content/general/concepts/information-technology/information-technology.md` adding in the "concepts" folder which is where all the other entries are located. 
> > 
> > Ok, I think I got it @KTom101. :)
> 
> Let me know if you have any trouble! I could always make the change on my end. 

Things are fine @KTom101. The checks have passed! :)

**Comment by KTom101** [comment]
Great!

**Pull request merged**